requirements/minimal_requirements.txt

```diff
@@ -0,0 +1,40 @@
+# we use this file to track minimal dependencies
+certifi==2017.4.17
+hydra-core==1.0.0
+lightly_utils~=0.0.0
+numpy==1.21.6
+python_dateutil==2.5.3
+requests==2.23.0
+six==1.10
+tqdm==4.57.0
+urllib3==1.25.3
+pydantic==1.10.5
+aenum==3.1.11
+pytorch_lightning==1.7.1
+torch==1.11.0
+torchvision==0.12.0
+pillow==7.1.2
+
+tox
+sphinx
+pylint
+pytest
+pytest-forked
+pytest-xdist
+pytest-mock
+responses
+docutils<=0.16
+sphinx-copybutton
+sphinx-design
+sphinx-gallery
+sphinx-tabs
+sphinx-reredirects
+sphinx_rtd_theme
+
+pre-commit
+opencv-python==4.4.0.46
```

Comment:
We should be able to drop this in the future. Opencv is only used in some old tests.

Reply:
Made an issue for it #1477